pvtoolsSrc/pvget.cpp

```diff
@@ -360,9 +366,6 @@ int MAIN (int argc, char *argv[])
                         "Option '-%c' requires an argument. ('" EXECNAME " -h' for help.)\n",
                         optopt);
                 return 1;
-            default :
-                usage();
-                return 1;
```

Comment:
Should this code retain the default case handling, to catch any future mis-matches between the options string passed into `getopt()` and the set of cases that the code actually handles? It might want to output a different error message though.

Reply:
I think the question mark will do that already?

Reply:
I'm not sure whether you're answering my question or not, I'll re-phrase it. What happens if someone adds several new letters to the `getopt()` string but forgets to add a new `case` statement for one of those letters; or removes one or more cases but forgets to remove the option letter(s) from the string? The code has then told `getopt()` that it supports an option, but nothing actually handles that if a user tries it.

Reply:
ah sorry, yes i understand. I think printing something about the given option not being supported (despite what getopt thinks) and then showing the usage would make sense.

Reply:
have added back in https://github.com/epics-base/pvAccessCPP/pull/195/commits/b593e3da157751d642747e1d8327ef38bb373830